pkg/apis/autoscaling/annotation_validation_test.go

```diff
@@ -40,29 +40,23 @@ func TestValidateScaleBoundAnnotations(t *testing.T) {
 	}, {
 		name:        "minScale is 0",
 		annotations: map[string]string{MinScaleAnnotationKey: "0"},
-		expectErr: &apis.FieldError{
-			Message: fmt.Sprintf("Invalid %s annotation value: must be an integer greater than 0", MinScaleAnnotationKey),
-			Paths:   []string{MinScaleAnnotationKey},
-		},
+		expectErr:   nil,
```

Comment:
Lets add tests for -1 on both of these since we are losing the failing edge case of 0.

Reply:
ok, added.